packages/react-dropdown/src/MultiDownshift.js

```diff
@@ -102,7 +98,7 @@ class MultiDownshift extends React.Component<Props, State> {
     const selectedItems = this.getSelectedItems();
     let newSelectedItems = [];
     if (selectedItem === null) {
-      newSelectedItems = this.clearItems();
```

Comment:
this method name looked like something one could call to clear the selected items, but it actually just returned an empty array that had to be manually assigned to the right property. I removed it to avoid confusion

Reply:
Is it working now when you call dowshifts `clearSelection`?